bladex/propeller.py

```diff
@@ -75,6 +75,6 @@ def display(self):
         """
         Display the propeller with shaft.
         """
-        display, start_display, add_menu, add_function_to_menu = init_display()
+        display, start_display, _, _ = init_display()
```

Comment:
In this case you can use
`display, start_display = init_display()[:2] `
to assign only the first 2 variables

Reply:
Thanks: modified!